tests/phpunit/elementor/includes/test-user.php

```diff
@@ -31,4 +32,73 @@ public function test_get_user_option_with_default__returns_value() {
 		// Assert.
 		$this->assertEquals( 'actual-value', $value );
 	}
+
+	public function test_is_user_notice_viewed__returns_false_on_new_website() {
+		// Arrange.
+		$user = $this->act_as_admin();
+
+		delete_user_meta( $user->ID, self::ADMIN_NOTICES_KEY );
+
+		// Act.
+		$is_viewed = User::is_user_notice_viewed( 'some-notice-id' );
+
+		// Assert.
+		$this->assertFalse( $is_viewed );
+	}
+
+	public function test_is_user_notice_viewed__BC__returns_true_when_viewed() {
```

Comment:
put the BC tests at the end, and maybe add a comment

Reply:
Done.